modules/security-context-constraints-psa-opting.adoc

```diff
@@ -10,22 +10,24 @@ You can enable or disable automatic pod security admission synchronization for m
 
 [IMPORTANT]
 ====
-Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
+Most namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
```

Comment:
i don't think any NSes that are in the payload, can have PSA syncing enabled.  The syncer has those NS names hardcoded into an ignore list.

was there one you're thinking of that can have syncing turned on?  I guess "openshift-operators" might be the solitary exception, but we explicitly label that NS as privileged anyway.

see: https://github.com/openshift/cluster-policy-controller/blob/523e4104074f88047de7d725d5c99ca4dbf8c167/pkg/psalabelsyncer/nsexemptions/nsexemptions.go#L9

Reply:
I did not have a specific namespace in mind. I think I just misunderstood some of the connections between label synchronization and some of the changes we made to the Catalog Source API.